### PR TITLE
MCW lab updates

### PR DESCRIPTION
**Pull request by tiagocostapt**
June 2023 lab updates.
@joelhulen

### CODE REVIEW

File: Hands-on lab/Before the HOL - Building the business migration case with Windows Server and SQL Server.md [hunk 1]
```diff
@@ -84,15 +85,15 @@ In this task, you will leverage a custom Azure Resource Manager (ARM) template t
 
 4. Fill in the required ARM template parameters.
     - Create a new **Resource group**.
-    - Set **Region** to `North Central US`
+    - Set **Region** to `North Central US`.
     - Specify a **Resource Name Base** (specify a base value that includes your initials to keep unique to prevent naming conflicts, for example, `tailspincp`).
     - Set `Azure Ad User Id` to the Azure AD `id` of the user that was previously copied from the Azure CLI.
     - Set the `Azure Ad User Login` to the Azure AD `userPrincipalName` that was previously copied from the Azure CLI.
-    - Select **Review + create**
+    - Select **Review + create**.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Before the HOL - Building the business migration case with Windows Server and SQL Server.md:93]: Hi!  Can we add an image here that follows the steps above please?  Red boxes around the fields you mentioned.

Comment (tiagocostapt) [reply Hands-on lab/Before the HOL - Building the business migration case with Windows Server and SQL Server.md:93]: Yes. I will add it.

Comment (DawnmarieDesJardins) [reply Hands-on lab/Before the HOL - Building the business migration case with Windows Server and SQL Server.md:93]: You're quick!  Thank you!  I did add a couple of commas to the doc, please make sure to download the latest version.  Moving on to the full lab now. :0)

File: Hands-on lab/HOL step-by-step - Building the business migration case with Windows Server and SQL Server.md [hunk 2]
```diff
@@ -308,41 +312,45 @@ In this exercise, you will go through the steps necessary to migrate Tailspin To
...
 
     ![The WideWorldImporters migration is shown in the Azure Portal with a migration status of succeeded.](images/wideworldimporters-migration-succeeded.png "WideWorldImporters migration showing status of succeeded")
 
-23. Within the Azure Portal, navigate to the **Azure SQL Managed Instance** that was created previously.
+24. Within the Azure Portal, navigate to the **Azure SQL Managed Instance** created previously.
 
-24. When the SQL Server database migration to Azure SQL MI has completed, you will see the **WideWorldImporters** database shown with an **Online** status.
+25. When the SQL Server database migration to Azure SQL MI has completed, you will see the **WideWorldImporters** database shown with an **Online** status.
 
     ![The Azure SQL Managed Instance resource is shown in the Azure Portal with the WideWorldImporters migration showing a Status of Online.](images/azure-portal-sql-mi-database-status-online.png "Azure SQL MI in Azure Portal showing the WideWorldImporters database in Online status")
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Building the business migration case with Windows Server and SQL Server.md:356]: Question for you.  In line 361, it says, "...use Azure Bastion to connect to the VM over Remote Desktop (RDP).  The acronym RDP...are we missing a word for the P, or is the acronym supposed to be RD??

Comment (DawnmarieDesJardins) [reply Hands-on lab/HOL step-by-step - Building the business migration case with Windows Server and SQL Server.md:356]: Apologies if this is a dumb question.  Can you please double check line 445 in the lab - is CD/CD correct, or should it be CI/CD?

VM has been created in Azure, Tailspin Toys will now be able to modify their Continuous Integration and Continuous Deployment (CD/CD) pipelines within....

File: Hands-on lab/Before the HOL - Building the business migration case with Windows Server and SQL Server.md (hunk 1 above)
Comment (tiagocostapt) [reply Hands-on lab/Before the HOL - Building the business migration case with Windows Server and SQL Server.md:93]: Done on June 11th commit.

File: Hands-on lab/HOL step-by-step - Building the business migration case with Windows Server and SQL Server.md (hunk 2 above)
Comment (tiagocostapt) [reply Hands-on lab/HOL step-by-step - Building the business migration case with Windows Server and SQL Server.md:356]: Fixed the CI/CD. Indeed it was wrong. Thanks for the catch! 
Regarding the RDP (line 361), I usually address that way "Remote Desktop (RDP)." I just want to say that we will remote into the VM's desktop using RDP. If you see the need to change that, ping me, and I will commit an update. It's a super easy fixe :)

**Pull request merged**